cpp/oneapi/dal/algo/kmeans_init/backend/cpu/compute_kernel_dense.cpp

```diff
@@ -35,7 +35,19 @@ template <typename Float, daal::CpuType Cpu>
 using daal_kmeans_init_dense_kernel_t =
     daal_kmeans_init::internal::KMeansInitKernel<daal_kmeans_init::defaultDense, Float, Cpu>;
 
```

Comment:
Name file does not match the method. I would remove `dense` from name.

Reply:
at this moment it implemented in another way.